ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```diff
@@ -219,7 +219,7 @@ objects:
       - name: HOME
         value: /tmp
       - name: MIRROR_BASE
-        value: registry.svc.ci.openshift.org/${NAMESPACE}/release
+        value: ${NAMESPACE}/release
```

Comment:
Hrm, I thought mirror base has to be fully qualified in order to use it in the release build?

Reply:
@smarterclayton the rehearsal failed like this in build, is that an infra thing?

```
 This system is not receiving updates. You can use subscription-manager on the host to register and assign subscriptions.
Package python-netaddr is obsoleted by python2-netaddr, trying to install python2-netaddr-0.7.19-5.el7.noarch instead
No package awscli available.
Error: Not tolerating missing names on install, stopping.
error: build error: running 'yum update -y &&     yum install --setopt=tsflags=nodocs -y     python-openstackclient ansible python-openstacksdk python-netaddr awscli &&     yum clean all && rm -rf /var/cache/yum/*' failed with exit code 1 
```

Reply:
I can undo the mirror stuff, just overzealous `sed`